t/allergens_tags.t

```diff
@@ -5,6 +5,7 @@ use utf8;
 
 use Test::More;
 use Log::Any::Adapter 'TAP';
+
```

Comment:
Inserting blanks before comments like this doesn't make much sense to me. Maybe we should consider changing the comment to static block comment (`##` by default) to keep it in place, or use `--noblanks-before-comments` to completely disable the feature.

Reply:
I added `--noblanks-before-comments` which seemed to fix that, thanks!